deployment/mesh-infra/storage/directpv/kustomization.yaml

```diff
@@ -10,10 +10,9 @@ resources:
 # 1. Direct PV. We generated the initial YAML file by running:
 #
 # $ cd nix && nix shell
-# $ kubectl directpv install --dry-run --image 'directpv:v3.2.2' > base.yaml
+# $ kubectl directpv install -o yaml > base.yaml
 #
-# Notice the Docker image version you specify above should be the
-# same as that of the `kubectl-directpv` tool in the Nix shell.
-# See also:
-# - https://github.com/minio/directpv/blob/master/docs/installation.md
-# - https://github.com/minio/directpv/blob/master/docs/cli.md
+# Notice our version of Direct PV (4.0.6) can only generate manifests
+# for x86_84/amd64. This is b/c they pull specific images for those
+# architectures. The `base.arm64.yaml` manifest fixes that by replacing
+# x86_84/amd64 images w/ arm64 ones. Diff the two files.
```

Comment:
It looks like we might have to pull together microK8s-specific manifests too

- https://github.com/minio/directpv/issues/11
- https://github.com/minio/directpv/issues/588

We should be able to take the output of the install command and then override the Kubelet path. Ideally we'd have x86_84 and arm64 variants...

Reply:
Actually this isn't an issue anymore with microK8s 1.27. In fact `/var/lib/kubelet` gets symlinked to `/var/snap/microk8s/common/var/lib/kubelet`.

```bash
$ ls -al /var/lib/kubelet
lrwxrwxrwx 1 root root 41 Jul  7 09:31 /var/lib/kubelet -> /var/snap/microk8s/common/var/lib/kubelet
```

Reply:
I was wrong. It is an issue actually. Read this

- https://github.com/minio/directpv/issues/816#issuecomment-1637071486

Reply:
Added MicroK8s manifest for x86-84. Notice DirectPV 4.0.6 does **not** support ARM64. See:
- https://github.com/minio/directpv/issues/817